README.md

```diff
@@ -1,2 +1,179 @@
-# Hello-world
-First Microverse project day 2 - Set up a 'Hello Microverse' project
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
```

Comment:
- [ ]  Kindly remove all the comments under the sections in the readme file. These comments are just to guide you on how to fill the readme file correctly.